### This is diff between two diffrent topics
### Compared it +content, most probably this is CLUS, it look OK
### CLHS http://clhs.lisp.se/Body/t_bit.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:bit.txt	2017-04-27 21:50:01.000000000 +0100
+++ saved-pages-clus/cl:types:bit.txt	2017-04-27 21:50:01.000000000 +0100
@@ -1,4 +1,16 @@
-BIT
-Please select which reference to BIT you intended
+Type BIT
 
-AccessorTypeGlossary Entry
\ No newline at end of file
+Supertypes
+bit, unsigned-byte, signed-byte, integer, rational, real, number, t
+Description
+The type bit is equivalent to the type (integer 0 1) and (unsigned-byte 1).
+\
\ No newline at end of file
